### PR TITLE
Modifying test_target_enter_data_malloced.. to use macros

### PR DESCRIPTION
**Pull request by tmh97**
Added macro usage, adjusted formatting issues, checks for offloading, tested and passed on GCC and Clang. Issue #10 *Still need to finish header*.

### CODE REVIEW

File: tests/4.5/target_enter_data/test_target_enter_data_malloced_array.c [hunk 1]
```diff
@@ -1,51 +1,66 @@
...
 
-void init(int **A){
+void init(int **A) {
  int i;
- *A = (int *)malloc(n*sizeof(int));
- if(NULL == *A) exit(-1);
- x=*A;
- for(i=0; i<n; i++){
+ *A = (int *) malloc(n*sizeof(int));
+ if (NULL == *A) exit(-1);
```
Comment (josemonsalve2) [inline tests/4.5/target_enter_data/test_target_enter_data_malloced_array.c]: We should consider an OMPVV_ERROR here as well as the exit. Or overpass the whole test and let the REPORT say that it failed.

File: tests/4.5/target_enter_data/test_target_enter_data_malloced_array.c [hunk 2]
```diff
@@ -1,51 +1,66 @@
...
+
 #include <stdio.h>
 #include <stdlib.h>
 #include <omp.h>
+#include "ompvv.h"
 
 // Test for OpenMP 4.5 target enter and target exit data with allocated arrays.
 
+// global variables
 int n=10, B[10];
```
Comment (josemonsalve2) [inline tests/4.5/target_enter_data/test_target_enter_data_malloced_array.c]: Any reason why n is not a `#define`?

Comment (tmh97) [reply tests/4.5/target_enter_data/test_target_enter_data_malloced_array.c]: good catch, will swap out.

File: tests/4.5/target_enter_data/test_target_enter_data_malloced_array.c (hunk 1 above)
Comment (tmh97) [reply tests/4.5/target_enter_data/test_target_enter_data_malloced_array.c]: Going to go with an OMPVV_ERROR message indicating it has failed

File: tests/4.5/target_enter_data/test_target_enter_data_malloced_array.c (hunk 2 above)
Comment (tmh97) [reply tests/4.5/target_enter_data/test_target_enter_data_malloced_array.c]: Seems it has to be a named variable, throwing issues when I try to use #define

**Pull request merged**